contents/articles/2016-07_pupil-service/index.md

```diff
@@ -0,0 +1,10 @@
+---
+ title: Pupil Service
+ date: Tue Jul 07 2016 09:47:48 GMT+0700 (ICT)
+ author: Pupil Dev Team
+ subtitle: Pupil Service is a new app in release v0.8.0...
+ ---
+
```

Comment:
Insert image of the Pupil Service icon this is in the web-media folder and also in pupil repo in deploy dir.